app/Views/public/support.php

```diff
@@ -31,71 +32,102 @@
                     <div class="support-progress__bullet">
                         <span class="support-progress__span">2</span>
                     </div>
-                    <div class="support-progress__check">
-                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-check-lg" viewBox="0 0 16 16">
-                        <path d="M12.736 3.97a.733.733 0 0 1 1.047 0c.286.289.29.756.01 1.05L7.88 12.01a.733.733 0 0 1-1.065.02L3.217 8.384a.757.757 0 0 1 0-1.06.733.733 0 0 1 1.047 0l3.052 3.093 5.4-6.425a.247.247 0 0 1 .02-.022Z"/>
-                        </svg>
-                    </div>
                 </div>
 
                 <div class="support-progress__step">
                     <p class="support-progress__name">Problema</p>
                     <div class="support-progress__bullet">
                         <span class="support-progress__span">3</span>
                     </div>
-                    <div class="support-progress__check">
-                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-check-lg" viewBox="0 0 16 16">
-                        <path d="M12.736 3.97a.733.733 0 0 1 1.047 0c.286.289.29.756.01 1.05L7.88 12.01a.733.733 0 0 1-1.065.02L3.217 8.384a.757.757 0 0 1 0-1.06.733.733 0 0 1 1.047 0l3.052 3.093 5.4-6.425a.247.247 0 0 1 .02-.022Z"/>
-                        </svg>
-                    </div>
                 </div>
             </div>
 
             <?php 
                 $response = session()->get('response');
                 if(isset($response)):
             ?>
-                <script type="module">
-                    Swal.fire({
-                        title: "<?= $response['title']; ?>",
-                        text: "<?= $response['message']; ?>",
-                        icon: "<?= $response['type']; ?>",
-                        confirmButtonColor: '#0174F6'
-                    });
-                </script>
+            <div id="alert-response"
+                data-title="<?= $response['title']; ?>"
+                data-message="<?= $response['message']; ?>"
+                data-type="<?= $response['type']; ?>"
+                style="display: none;"></div>
             <?php endif;?>
 
-            <form class="support-form" id="support-form" method="POST" action="/email/soporte"> 
-                <fieldset class="support-form__fieldset" id="soporte-fieldset-2">
-                    <legend class="support-form__legend">Fieldset</legend>
-                    <p>This fieldset is not available until ticket KIN-84 is completed</p>
-                    <a class="support-form__btn" id="btn-next-1">Siguiente</a>
+            <form class="support-form" id="support-form" action="/email/soporte" method="POST" novalidate> 
+                <fieldset class="support-form__fieldset support-form__step" data-step="1">
```

Comment:
```suggestion
                <fieldset class="support-form__fieldset active support-form__step" data-step="1">
```
we add the .active class to de first fieldset by default